client/actions/index.js

```diff
@@ -0,0 +1,15 @@
+export const REQUEST_ASSESSMENT_STATUSES = 'REQUEST_ASSESSMENT_STATUSES'
```

Comment:
Hi, this looks good but could we rename the file to be `assessmentStatus.js`. Thanks!